apps/heft/src/schemas/sass.schema.json

```
@@ -32,6 +32,15 @@
       "description": "Determines if export values are wrapped in a default property, or not."
     },
 
+    "fileExtensions": {
+      "type": "array",
+      "description": "Files with these extensions will pass through the Sass transpiler for typings generation.",
+      "items": {
+        "type": "string",
+        "pattern": "[^\\\\]"
```

Let's update this pattern to something like

```suggestion
        "pattern": "^\\.[A-z0-9-._]*[A-z0-9-]+$"
```

This says: 
- Must start with a `.`
- Other characters must be alphanumeric, `-`, `_`, or `.` (except for the last character)

That doesn't cover 100% of legal file extensions, but it should be pretty close.

It'd also be good to update [this expression](https://github.com/microsoft/rushstack/blob/master/apps/heft/src/schemas/typescript.schema.json#L73). Might be a good idea to extract this to a resource in a shared schema file.